pkg/kamailio/centos/7/kamailio.spec

```diff
@@ -214,8 +214,8 @@ json string handling and RPC modules for Kamailio.
 %package    kazoo
 Summary:    Kazoo middle layer connector support for Kamailio
 Group:      System Environment/Daemons
-Requires:   libuuid, rabbitmq-c, json-c, libevent, kamailio = %ver
-BuildRequires:  libuuid-devel, rabbitmq-c-devel, json-c-devel, libevent-devel
+Requires:   libuuid, librabbitmq >= 0.8.0, json-c, libevent, kamailio = %ver
+BuildRequires:  libuuid-devel, librabbitmq-devel >= 0.8.0, json-c-devel, libevent-devel
```

Comment:
Module can be build but later cannot be used if used rabbitmq-c lib 0.8.0
This PR is fix it